### PR TITLE
Ahhhhmeds change

### PR DESCRIPTION
**Pull request by Ahhhhmed**
Please review this, @casey-collab

### CODE REVIEW

File: english/dance/bake-a-cake/bake-a-cake.md [hunk 1]
```diff
@@ -9,3 +9,5 @@ Do you:
 [Sleep?](../../sleep/marshmallow.md)
 
 [Sing a song?](../../sing-song/sing.md)
+
+[Drink some coffee?](../../coffe/coffee.md)
```
Comment (casey-collab) [inline english/dance/bake-a-cake/bake-a-cake.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Ahhhhmed** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :heavy_check_mark:

**Pull request merged**